packages/web3-eth-contract/src/contract.ts

```diff
@@ -446,11 +446,7 @@ export class Contract<Abi extends ContractAbi>
 	);
 	public constructor(
 		jsonInterface: Abi,
-		addressOrOptionsOrContext?:
-			| Address
-			| ContractInitOptions
-			| Web3ContractContext
-			| Web3Context,
+		addressOrOptionsOrContext?: Address | ContractInitOptions | Web3ContractContext | Web3Context,
```

Comment:
will be very helpful to not include such changes , thanks,

Reply:
Sorry, those changes came after I saved the file. They are made by prettier according to the current configuration. 
So, either the file was saved early without applying prettier formatting. Or, there is something different on my machine. For now, I will revert those changes back.